.config/mkdocs.yml

```diff
@@ -134,7 +134,7 @@ nav:
   - Practice areas:
       - About practice areas: practice-areas/about-practice-areas.md
       - Accessibility practice area: practice-areas/accessibility-practice-area.md
-      - Design and research:
+      - Design and research: practice-areas/design-and-research/design-research-methods.md
           - Audience and outcomes guide: practice-areas/design-and-research/audiences-and-outcomes-guide.md
           - Content audit guide: practice-areas/design-and-research/content-audit-guide.md
           - Content modeling guide: practice-areas/design-and-research/content-modeling-guide.md
```

Comment:
@ContentBrewmaster I believe this needs to be moved one row below `- Design and research:` and indented. It does not need to have a title as it will default to using the top level title. So it would it look like this:

```
      - Design and research:
          - practice-areas/design-and-research/design-research-methods.md 
          - Audience and outcomes guide: practice-areas/design-and-research/audiences-and-outcomes-guide.md
          - Content audit guide: practice-areas/design-and-research/content-audit-guide.md
          - Content modeling guide: practice-areas/design-and-research/content-modeling-guide.md
```

That will resolve the mkdocs error. The above is the way to make the parent menu item also link a page and not just control the expand/collapse of the sub menu.